doc/developer/logging.rst

```diff
@@ -361,6 +355,51 @@ FRR library helper formats
 
    (The output is aligned to some degree.)
 
+FRR daemon specific formats
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following formats are only available in specific daemons, as the code
+implementing them is part of the daemon, not the library.
+
+zebra
+"""""
+
+.. frrfmt:: %pZN (struct route_node *)
+
+   Print information for a RIB node, including zebra-specific data.
+
+   :frrfmtout:`::/0 src fe80::/64 (MRIB)` (``%pZN``)
+
+   :frrfmtout:`1234` (``%pZNt`` - table number)
+
+bgpd
+""""
+
+.. frrfmt:: %pBD (struct bgp_dest *)
+
+   Print prefix for a BGP destinatin.
+
+   :frrfmtout:`fe80::1234/64`
+
+pimd/pim6d
+""""""""""
+
+.. frrfmt:: %pPA (pim_addr *)
+
+   Format IP address according to IP version (pimd vs. pim6d) being compiled.
+
+   :frrfmtout:`fe80::1234` / :frrfmtout:`10.0.0.1`
+
+   :frrfmtout:`*` (``%pPAs`` - replace 0.0.0.0/:: with star)
+
+.. frrfmt:: %pSG (pim_sgaddr *)
+
```

Comment:
Since for (S,G), now we use Please remove the below statement which is defined under %pPSG4
"""
This is (S,G) output for use in pimd. (Note prefix_sg is not a prefix
"subclass" like the other prefix_* structs.)
"""

Reply:
`prefix_sg` is still used in the zebra VXLAN code, that's why it's still there